.github/workflows/n-minus-one-install.yml

```diff
@@ -0,0 +1,18 @@
+name: "N-1 on example"
+on: 
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  
+jobs:
+  test_n_minus_one: #test that an older, pinned playwright/test dependency is able to use npx playwright install
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '16'
+    - run: cd examples/todomvc; npm install
+    - run: cd examples/todomvc; npx playwright install
```

Comment:
You are missing the `npx playwright install-deps`, so things are not supposed to work.

Reply:
i am facing same issue and does not how to resolve it my docker container is keep failing due to this error i have done all the step still non of them worked.

please let know if there is any specific solutions to solve this.